include/fluent-bit/flb_langinfo.h

```diff
@@ -140,5 +143,7 @@ static inline const char *nl_langinfo(nl_item item)
         return "";
     return lc_time_c[item];
 }
+#else
+#include <langinfo.h>
```

Comment:
Moved this downwards as the above condition is to complex to add an extra negation